docs/apis/subsystems/roles.md

```diff
@@ -0,0 +1,612 @@
+    1. `mod/hotpot:grade` - (grade? and) regrade
+    1. `mod/hotpot:deleteattempt` - deletes attempts
+9. Label
+    1. none
+10. Lams
+    1. `mod/lams:participate` - original student
+    1. `mod/lams:manage` - original teacher
+11. Lesson
+    1. `mod/lesson:edit` - add and edit pages
+    1. `mod/lesson:manage` - view student attempts
+12. #Quiz
+     1. `mod/quiz:grade` - comment, override grade, manual grade
+     1. `mod/quiz:preview` - previews the quiz
+     1. `mod/quiz:viewreports` - view quiz result reports
+     1. `mod/quiz:manage` - add/delete/move (up or down) questions for a quiz
+     1. `mod/quiz:attempt` - attempt the quiz--[Tim Hunt](https://docs.moodle.org/dev/User:Tim_Hunt)
+13. Resource
+14. Scorm
+    1. `mod/scorm:viewgrades`
+15. Survey
+    1. `mod/survey:download` - downloads server result
+    1. `mod/survey:participate` - participate/ do survey
+    1. `mod/survey:readresponses` - read all user's response
+16. Wiki
+    1. `mod/wiki:participate` - original student, meaning depends on type and course setting
+    1. `mod/wiki:manage` - original teacher, manages assigned group; `moodle/site:accessallgroups` is needed to manage all groups
+    1. (Waiting on new wiki)
+17. Workshop
+    1. `mod/workshop:participate` - original student, allows user to submit and assess
+    1. `mod/workshop:manage` - original teacher, user can manage others
+    1. (Waiting on new Workshop)
+
+### Enrolment-level Capabilities
+
+The naming convention for capabilities that are specific to enrolment is `enrol/enrol_name:capability`. The enrolment capabilities are defined in `enrol/enrol_name/db/access.php`.
+
+1. Authorize.net Payment Gateway
+   1. `enrol/authorize:managepayments` - manage user payments, capture, void, refund, delete etc.
+
+   1. Blocks
+1. `activity_modules`
+   1. None
+1. `admin`
+1. `admin_2`
+1. `admin_bookmarks`
+1. `blog_menu`
+1. `blog_tags`
+1. `calendar_month`
+1. `calendar_upcoming`
+1. `course_list`
+1. `course_summary`
+1. `glossary_random`
+1. `html`
+1. `loancalc`
+1. `login`
+1. `messages`
+1. `news_items`
+1. `online_users`
+1. `participants`
+1. `quiz_results`
+1. `recent_activity`
+1. `rss_client`
+    1. `block/rss_client:createprivatefeeds`
+    2. `block/rss_client:createsharedfeeds`
+    3. `block/rss_client:manageownfeeds`
+    4. `block/rss_client:manageanyfeeds`
+1. `search`
+1. `search_forums`
+1. `section_links`
+1. `site_main_menu`
+1. `social_activities`
+
+### Questions
+
+I am adding question categories here because they seem to have been forgotten in the whole scheme of things since having been removed from the quiz module itself. I've made a suggestion on how these could be handled in [MDL-6118](https://tracker.moodle.org/browse/MDL-6118).
+
+: See [this forum thread](http://moodle.org/mod/forum/discuss.php?d=51143) for a discussion about the current problems wth publishing question categories.[Tim Hunt](https://docs.moodle.org/dev/User:Tim_Hunt) 18:50, 8 August 2006 (WST)
+
```

Comment:
Remove this. It's irrelevan tand very out-of-date.

Better to give people the information they need to find this out for themslves

```suggestion
```